WordPressAuthenticatorTests/Authenticator/WordPressAuthenticatorTests.swift

```diff
@@ -38,21 +45,352 @@ class WordPressAuthenticatorTests: XCTestCase {
         url = WordPressAuthenticator.baseSiteURL(string: punycode)
         XCTAssert(url == punycode)
     }
-
+    
     func testEmailAddressTokenHandling() {
         let email = "example@email.com"
         let loginFields = LoginFields()
         loginFields.username = email
         WordPressAuthenticator.storeLoginInfoForTokenAuth(loginFields)
-
+        
         var retrievedLoginFields = WordPressAuthenticator.retrieveLoginInfoForTokenAuth()
         var retrievedEmail = retrievedLoginFields.username
         XCTAssert(email == retrievedEmail, "The email retrived should match the email that was saved.")
-
+        
         WordPressAuthenticator.deleteLoginInfoForTokenAuth()
         retrievedLoginFields = WordPressAuthenticator.retrieveLoginInfoForTokenAuth()
         retrievedEmail = retrievedLoginFields.username
-
+        
         XCTAssert(email != retrievedEmail, "Saved loginFields should be deleted after calling deleteLoginInfoForTokenAuth.")
     }
+    
+    func testAuthenticatorInitHasCorrectProperties() {
+        //COME BACK TO THIS
+        let authenticator = WordPressAuthenticator.shared
```

Comment:
I have gone ahead and changed the access control for the WPAuth class init to be internal so that I could initialize a mock of the authenticator instead of using the shared.  

Thanks for the suggestion!